hack/ec-opa.sh

```diff
@@ -19,4 +19,5 @@
 # by EC, e.g. ec.oci.image_files, a custom version of OPA is required.
 set -euo pipefail
 cd "$(dirname "$0")/.."
-exec go run github.com/enterprise-contract/ec-cli opa "$@"
+make ec-bin
```

Comment:
I think if the target is phony, it will want to build it every time, which is defeating the purpose. I think you need a non-phony target that is the filename, or do a check here in bash to see if the bin/ec exists already, something like  `[ -x bin/ec ] || make ec-bin`

Reply:
Thank you! Updated.